compiler/rustc_borrowck/src/type_check/mod.rs

```diff
@@ -1075,7 +1075,7 @@ impl<'a, 'tcx> TypeChecker<'a, 'tcx> {
                 proj,
```

Comment:
is the `if let ty::Alias` fast path necessary with the new solver or could we disable it with the new solver